pennylane/ops/qubit/non_parametric_ops.py

```diff
@@ -181,6 +183,8 @@ class PauliX(Observable, Operation):
 
     basis = "X"
 
+    _queue_category = "_ops"
```

Comment:
Is this always correct because when I measure X, the op itself is not queued but the measurement process?

Reply:
When you measure X, the measurement "owns" the observable.  If the operator has an "owner", then it is not processed into `_ops`.  

The problem is with observables that don't have owners.  Many un-owned tensors are queued during hamiltonian manipulation. They currently don't get processed into `_ops` because they don't inherit from `Operation`. To maintain identical behaviour without severely increasing maintenance of hamiltonian functionality, we need a way to denote objects that we never want processed into `_prep`, `_ops`, or `_measurements`.

Since `Observable` behaviour overrides `Operation` behavior for the pauli ops, we need to override the behaviour to once again allow pauli ops to queue.